app/code/Magento/MediaGalleryUi/Test/Mftf/Test/AdminStandaloneMediaGalleryDisabledTest.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminMediaGalleryDisabledPageTest">
+        <annotations>
+            <features value="MediaGallery"/>
+            <useCaseId value="https://github.com/magento/adobe-stock-integration/issues/1760"/>
+            <title value="Standalone Media Gallery Page should return 404 if Media Gallery is disabled"/>
+            <stories value="#1760 Media Gallery Page opened successfully if Enhanced Media Gallery disabled"/>
+            <testCaseId value="https://studio.cucumber.io/projects/131313/test-plan/folders/1337102/scenarios/5106786"/>
+            <description value="Standalone Media Gallery Page should return 404 if Media Gallery is disabled"/>
+            <severity value="CRITICAL"/>
+            <group value="media_gallery_ui"/>
```

Comment:
Please intorduce a new `media_gallery_ui_disabled` suite that will not enable the new media gallery (that suite does not need any before/after actions actually)